packages/action-button/src/spectrum-action-button.css

```diff
@@ -96,6 +108,74 @@ governing permissions and limitations under the License.
             var(--spectrum-actionbutton-focus-indicator-gap)
     );
 }
+:host([selected]) {
+    --mod-actionbutton-background-color-default: var(
```

Comment:
@pfulton the component specifically giving a value to the `--mod-` custom properties seems like it would break the contract that these can be customized from the outside by consumers. While this could be customized via `<sp-action-button style="--mod-actionbutton-background-color-default: ...">` the previously available approach of `<div style="--mod-actionbutton-background-color-default: ..."><sp-action-button>` to customize a while batch of Buttons is no longer possible.

Reply:
@pfulton correct me if I'm wrong here, but this is still possible, a different mod variable would just be used. This is for when the action button is selected, so you'll see that this variable is able to be overwritten by using `--mod-actionbutton-background-color-default-selected` and if that variable is not set it falls back to the selected value. This is our new approach to allow for some more variant/state specific mods while still keeping out CSS clean. Many of our recent components have implemented this approach

Reply:
@mlogsdon18 Well said. Another way to put this is that the default background color mod should now be leveraging the default-selected variant of that same mod. It's a way to cut down on unnecessary selectors when state changes.

Reply:
One last thought on this to help clarify the approach, the contract is maintained for sp-action-button to set --mod-actionbutton-background-color-default. The only time that variable will not work is when the selected state is active, in which case we expect to use --mod-actionbutton-background-color-default-selected instead anyway. This is how it works today except that css property is overridden and that property is being remapped to a different mod. This instead leverages 1 css property definition and just updates which variable it calls based on state. Let us know if there are any questions or clarifications we can make beyond this!

Reply:
Thanks for the clarification!